src/NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestTwoWayCapableLayoutRenderer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Http.Features;
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// Indicates if the server can upgrade this request to an opaque, bidirectional stream.
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-request-two-way-capable}
+    /// </remarks>
+    [LayoutRenderer("aspnet-request-two-way-capable")]
```

Comment:
Any reason why the type-alias doesn't use any words from the summary-docs ? https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.features.ihttpupgradefeature.isupgradablerequest